x-pack/plugins/watcher/public/sections/watch_edit/components/threshold_watch_edit_component.tsx

```diff
@@ -461,11 +484,25 @@ const ThresholdWatchEditUi = ({
               button={
                 <EuiExpression
                   description={`${watch.groupBy === 'all' ? 'over' : 'grouped over'}`}
-                  value={groupByTypes[watch.groupBy].text}
-                  isActive={groupByPopoverOpen}
+                  value={`${groupByTypes[watch.groupBy].text} ${
+                    watch.groupBy === 'all'
```

Comment:
i think there's a constants file that can probably be used here instead of hard-coding `all` - `group_by_types.ts`